packages/flutter/lib/src/widgets/image.dart

```diff
@@ -399,6 +399,32 @@ class Image extends StatefulWidget {
   /// In the case where the network image is on the Web platform, the [cacheWidth]
   /// and [cacheHeight] parameters are ignored as the web engine delegates
   /// image decoding to the web which does not support custom decode sizes.
+  ///
+  /// ### Cross-origin images on Web
```

Comment:
It's more correct to refer to the "Same-origin policy" and link to https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy